plugins/woocommerce-blocks/assets/js/blocks/cart/inner-blocks/cart-order-summary-shipping/shipping-rate-selector.tsx

```diff
@@ -28,31 +15,8 @@ export const ShippingRateSelector = (): JSX.Element => {
 			</legend>
 			<ShippingRatesControl
 				className="wc-block-components-totals-shipping__options"
-				noResultsMessage={
-					<>
-						{ hasCompleteAddress && (
-							<NoticeBanner
-								isDismissible={ false }
-								className="wc-block-components-shipping-rates-control__no-results-notice"
-								status="warning"
-							>
-								{ createInterpolateElement(
-									sprintf(
-										// translators: %s is the address that was used to calculate shipping.
-										__(
-											'No delivery options available for <strong>%s</strong>. Please verify the address is correct or try a different address.',
-											'woocommerce'
-										),
-										formatShippingAddress( shippingAddress )
-									),
-									{
-										strong: <strong />,
-									}
-								) }
-							</NoticeBanner>
-						) }
-					</>
-				}
+				// No longer needed as we display notice in the address panel instead.
+				noResultsMessage={ <></> }
```

Comment:
Could this prop be made optional in `ShippingRatesControl`?

Reply:
Yep definitely!